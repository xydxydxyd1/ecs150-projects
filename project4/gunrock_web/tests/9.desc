Run ds3cat a simple image
