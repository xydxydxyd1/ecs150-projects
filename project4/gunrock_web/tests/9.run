./ds3cat tests/disk_images/a.img 3
