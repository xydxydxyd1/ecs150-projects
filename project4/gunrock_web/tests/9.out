File blocks
4

File data
file contents
